components/script/dom/htmllinkelement.rs

```diff
@@ -256,10 +255,22 @@ impl HTMLLinkElement {
 
         let referrer_policy = match self.RelList().Contains("noreferrer".into()) {
             true => Some(ReferrerPolicy::NoReferrer),
-            false => None,
+            false => document.get_referrer_policy(),
+        };
+
+        let request = RequestInit {
+            url: url.clone(),
+            type_: RequestType::Script,
+            destination: Destination::Style,
```

Comment:
The spec says destination should be the empty string, but the table at https://fetch.spec.whatwg.org/#concept-request-type disagrees. I've filed https://github.com/whatwg/html/issues/1856.